src/Python/tests/tests.py

```diff
@@ -381,15 +434,14 @@ def test_metadata(self):
                               "name":"Behavioural Genomics" },
                        "who":"Firstname Lastname",
                        "timestamp":"2012-04-23T18:25:43.511Z",
-                       "temperature":{ "experiment":22,
-                                       "cultivation":20, "units":"C" },
-                       "humidity":{ "value":40, "units":"%" },
+                       "temperature":23.8,
+                       "humidity":40.3,
                        "dish":{ "type":"petri", "size":35, "units":"mm" },
                        "food":"none",
                        "media":"agarose",
                        "sex":"hermaphrodite",
                        "stage":"adult",
-                       "age":"18:25:43.511",
+                       "age":18.511,
```

Comment:
Aha, this must be the origin of my `18.511` figure from above as well.  I'll change to something around 33 hours instead.